src/components/Pages/SetupPage/SetupPage.tsx

```diff
@@ -299,6 +337,68 @@ export const SetupPage = () => {
     )
   }
 
+  if (isMobile) {
```

Comment:
I'd like to move these states to different files on a different PR
- Randomize
- Form
- Mobile Randomize
- Mobile Form

Reply:
Also there is a bunch of repeated code I'd like to abstract. Not on this PR because the diff would be to ugly.